frostsnap_core/src/coordinator.rs

```diff
@@ -786,39 +919,49 @@ pub enum KeyGenState {
     },
 }
 
-#[derive(Clone, Debug, bincode::Encode, bincode::Decode, serde::Serialize, serde::Deserialize)]
-pub struct CoordinatorFrostKey {
-    frost_key: SharedKey,
-    device_to_share_index: BTreeMap<DeviceId, Scalar<Public, NonZero>>,
-    key_name: String,
+#[derive(Clone, Debug, bincode::Encode, bincode::Decode, PartialEq)]
+pub struct CoordAccessStructure {
+    app_shared_key: Xpub<SharedKey>,
```

Comment:
Initial reaction was that Xpub might be a bit out of place for future beyond-bitcoin purposes, but other applications can just ignore the chaincode if they don't need it

Reply:
Yes. We're all in on bip32 so it has to be there.